### PR TITLE
Add vim setup

### PR DESCRIPTION
**Pull request by thebluber**

### CODE REVIEW

File: configs/vimrc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+execute pathogen#infect()
```
Comment (jeanmartin) [inline configs/vimrc]: wrong!

File: configs/vimrc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+call pathogen#helptags()
```
Comment (jeanmartin) [inline configs/vimrc]: no

File: configs/vimrc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+call pathogen#infect()
```
Comment (jeanmartin) [inline configs/vimrc]: wrong again!

File: configs/vimrc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+set term=xterm-256color
```
Comment (jeanmartin) [inline configs/vimrc]: this is stupid

Comment (thebluber) [reply configs/vimrc]: Fuck you too!

File: configs/vimrc (hunk 1 above)
Comment (thebluber) [reply configs/vimrc]: make it better you jerk!

File: raspberry_setup.sh [hunk 5]
```diff
@@ -10,8 +10,7 @@ sudo apt-get install -y \
     wget \
     sl
 
-# vimrc
-wget 
+chmod u+x vim_setup.sh && ./vim_setup.sh
```
Comment (jeanmartin) [inline raspberry_setup.sh]: how is this supposed to work?

Comment (SimonMiaou) [reply raspberry_setup.sh]: Somebody didn't follow

**Review by jeanmartin — APPROVED** [review]

**Pull request merged**